app/controllers/assignment_invitations_controller.rb

```diff
@@ -35,11 +35,7 @@ def accept
     else
       create_submission do
         GitHubClassroom.statsd.increment("exercise_invitation.accept")
-        if current_submission.starter_code_repo_id
-          redirect_to setup_assignment_invitation_path
-        else
-          redirect_to success_assignment_invitation_path
-        end
+        redirect_to success_assignment_invitation_path
```

Comment:
@benemdon Could you test this locally on a repo with starter code and without import resiliency enabled?

Reply:
## With Starter code
![jul-18-2018 11-22-36](https://user-images.githubusercontent.com/11095731/42900182-1adad9c6-8a7d-11e8-9e10-6cf8c9a083d4.gif)

## Without starter code
![jul-18-2018 11-23-41](https://user-images.githubusercontent.com/11095731/42900195-20838a4e-8a7d-11e8-90e7-7742230a4490.gif)